web/client/codechecker_client/cmd/store.py

```diff
@@ -857,8 +859,12 @@ def main(args):
                       sizeof_fmt(MAX_UPLOAD_SIZE))
             sys.exit(1)
 
+        b64zip = ""
         with open(zip_file, 'rb') as zf:
             b64zip = base64.b64encode(zf.read()).decode("utf-8")
+        if len(b64zip) == 0:
```

Comment:
I only get an empty zip size when I try to store a directory which doesn't exist.  In this case I will get an exception before these statements.
So my question is that how it is possible that the zip size is empty?

Reply:
If for some reason we created an empty zip file because of an error in the assemble part or the zip file content was emptied between the assemble and the read before the send.
I do not mind extra error checking.